### ISSUE TITLE
Namespace not recognized if there is empty line after <?php

### ISSUE THREAD

**Issue by juliardi** [Gert-dev/php-integrator-autocomplete-plus#25]
When my file is started like this
```php
 <?php
 namespace Foo\Bar;
```
it appears in autocompletion. But, if the file started like this
```php
 <?php
 //empty line here
 namespace Foo\Bar;
```
it doesn't appear. Please fix this issue.
Great package btw, it makes Atom feels like an IDE.
Sorry for bad english

**Comment by Gert-dev** [comment]
Thanks for your kind words. Do you have a more complete example for me to look at? Neither newlines nor comments should pose a problem, I'm successfully using the same thing in all my code bases.

**Comment by juliardi** [comment]
Yes, i mean the class name doesn't show up in the class list during autocompletion if there is a newline.
I use Atom to develop in Laravel. Maybe you can try it yourself so you can understand the problem.

**Comment by juliardi** [comment]
Here is some screenshoot of this issue
1. No Empty Line
![no empty line](https://cloud.githubusercontent.com/assets/4223302/11618478/fa3cbdbe-9ccd-11e5-8780-cc45d659e7c9.png)
Classname show up in the class list
![autocompletion](https://cloud.githubusercontent.com/assets/4223302/11618481/fa40a122-9ccd-11e5-9b7f-32c54a09db63.png)
2. Empty Line
![empty line](https://cloud.githubusercontent.com/assets/4223302/11618479/fa3e1d80-9ccd-11e5-821d-1db331ec7459.png)
Classname doesn't show up
![no autocompletion](https://cloud.githubusercontent.com/assets/4223302/11618480/fa3eec7e-9ccd-11e5-8796-fd6183f60d48.png)

Also I don't know why, but some of my class in app/ directory didn't show up either

**Comment by Gert-dev** [comment]
Do the classes that aren't showing up happen to have the same name (i.e. are there classes in different namespaces that share the same name), e.g. `Foo\A` and `Bar\A`? If this is the case, this could be related to #15.

I'm also still trying to reproduce the newline thing, I don't have it in any of my projects. Does this also happen with a simple Laravel app or specifically in yours (if it's the latter, is it an open source codebase so I can examine it)?

**Comment by juliardi** [comment]
Do the classes that aren't showing up happen to have the same name?
No, they are in the same namespace so it's not possible. I have 7 class in my App\Models namespace, but none of them are showing up in the autocompletion even when i type the fully qualified class name.

Does this also happen with a simple Laravel app or specifically in yours?
I have try to create a new project of Laravel and it also happened. Even the Illuminate package of Laravel itself didn't show up.

**Comment by Gert-dev** [comment]
Ok, thanks for the information. This sounds almost certainly like a bug so I'll try again to reproduce it.

**Comment by Gert-dev** [comment]
Wanted to ping you and see if you still had this issue; in 0.4.0 the parsing on the PHP side became entirely different, did this fix anything for you? Additionally, if it didn't, there will be some more changes in the code detecting this in the upcoming 0.6.0.

**Comment by juliardi** [comment]
I'm sorry, I can't confirm if the problem been fixed or not. I'll be busy till next week, so I'll take a look when I have free time. Thank you for your concern over this problem.

**Comment by juliardi** [comment]
OK, I've confirmed that the problem has been fixed on version 0.5.1. Thank you for your concern and good job. Please remove the bug label in this issue.

**Comment by Gert-dev** [comment]
Good to hear, thanks for sticking with me!
